plugins/backend/catalog-backend-module-aws/src/providers/AWSEBSVolumeProvider.ts

```diff
@@ -92,51 +93,57 @@ export class AWSEBSVolumeProvider extends AWSEntityProvider {
       dynamicAccountConfig,
     );
 
-    const volumes = await ec2.describeVolumes({});
-
-    for (const volume of volumes.Volumes || []) {
-      const volumeId = volume.VolumeId;
-      if (!volumeId) continue;
-
-      const arn = `arn:aws:ec2:${region}:${accountId}:volume/${volumeId}`;
-      const arnParts = arn.split(':');
-      const resourceParts = arnParts[5].split('/');
-      const ebsVolumeId = resourceParts[3];
+    let nextToken: string | undefined = undefined;
+    do {
+      const volumes: DescribeVolumesCommandOutput = await ec2.describeVolumes({
+        NextToken: nextToken,
+      });
+      for (const volume of volumes.Volumes || []) {
+        const volumeId = volume.VolumeId;
+        if (!volumeId) continue;
+
+        const arn = `arn:aws:ec2:${region}:${accountId}:volume/${volumeId}`;
+        const arnParts = arn.split(':');
+        const resourceParts = arnParts[5].split('/');
+        const ebsVolumeId = resourceParts[3];
+
+        const consoleLink = createEbsVolumeConsoleLink(region, ebsVolumeId);
```

Comment:
There is a library that does this.

link2aws.ARN(arn).consoleLink

Reply:
doesn't do it for ebs volumes

Reply:
ah ok. that makes sense then.